core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -850,6 +869,18 @@ private[history] class AppListingListener(log: FileStatus, clock: Clock) extends
         fileSize)
     }
 
+    def applicationStatus : Option[String] = {
+      if (startTime.getTime == -1) {
+        Some("<Not Started>")
+      } else if (endTime.getTime == -1) {
+        Some("<In Progress>")
```

Comment:
I don't think these should have brackets

Reply:
Agree with you @ajbozarth